sys-devel/elftoolchain/elftoolchain-0.7.1-r2.ebuild

```diff
@@ -36,27 +36,23 @@ src_prepare() {
 	sed -i -e "s@readelf@$(tc-getREADELF)@" common/native-elf-format || die
 }
 
-_bmake() {
-	bmake \
-		AR="$(tc-getAR)" \
-		CC="$(tc-getCC)" \
-		LD="$(tc-getLD)" \
-		RANLIB="$(tc-getRANLIB)" \
-		"$@" || die
+src_configure() {
+	tc-export AR CC LD RANLIB
+	default
```

Comment:
does this thing actually invoke `econf`? :thinking:

Reply:
Ah no it doesn't, I'll drop the `default` call